README.md

```diff
@@ -26,3 +26,5 @@ Please complete the following by November 1st. This should be the next time we w
 * Trey Watford is Bester
 * Will Dixon Is better
 * Chris Bracey Is betterer
+* Kyle Heath Is bettererer
```

Comment:
This is what is confusing. Did you merge it or did you add this line manually? Adding manually makes it worse and needs to be merged instead. This is showing that this didn't exist before, but it does.